managed/models.go

```diff
@@ -357,3 +357,26 @@ type TelemetryProvider struct {
 	GoogleCloudSpec *GCPServiceAccount `tfsdk:"googlecloud_spec"`
 	IsValid         types.Bool         `tfsdk:"is_valid"`
 }
+
+type LogConfig struct {
+	LogMinDurationStatement types.Int64  `tfsdk:"log_min_duration_statement"`
+	DebugPrintPlan          types.Bool   `tfsdk:"debug_print_plan"`
+	LogConnections          types.Bool   `tfsdk:"log_connections"`
+	LogDisconnections       types.Bool   `tfsdk:"log_disconnections"`
+	LogDuration             types.Bool   `tfsdk:"log_duration"`
+	LogErrorVerbosity       types.String `tfsdk:"log_error_verbosity"`
+	LogStatement            types.String `tfsdk:"log_statement"`
+	LogMinErrorStatement    types.String `tfsdk:"log_min_error_statement"`
+	LogLinePrefix           types.String `tfsdk:"log_line_prefix"`
+}
+
+type DbQueryLoggingConfig struct {
+	AccountID       types.String `tfsdk:"account_id"`
+	ProjectID       types.String `tfsdk:"project_id"`
+	ClusterID       types.String `tfsdk:"cluster_id"`
```

Comment:
Why do we need both cluster id and cluster name? is cluster id only not sufficient?

Reply:
@arishta-yb cluster name makes it easier for the user to view the state file. Is it okay to have only `cluster_id` as the `Required` field and `cluster_name` as the `Computed` field?

Reply:
Discussed with Siddarth.
I have removed `cluster_name` and only taking `cluster_id` as mandatory field.